eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/configuration/ImportDbsnpVariantsJobConfigurationFailValidationStepTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.configuration;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.ac.ebi.eva.accession.dbsnp.parameters.InputParameters;
+import uk.ac.ebi.eva.accession.dbsnp.test.BatchTestConfiguration;
+import uk.ac.ebi.eva.accession.dbsnp.test.TestConfiguration;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {BatchTestConfiguration.class, TestConfiguration.class})
+@TestPropertySource("classpath:validate-contigs.properties")
+public class ImportDbsnpVariantsJobConfigurationFailValidationStepTest {
+
+    @Autowired
+    private InputParameters inputParameters;
+
+    @Autowired
+    private JobLauncherTestUtils jobLauncherTestUtils;
+
+    @Test
+    @DirtiesContext
+    public void executeJobTrueForceImport() throws Exception {
+        inputParameters.setForceImport("true");
+        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
+        assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
+    }
+
+    @Test
+    @DirtiesContext
+    public void executeJobFalseForceImport() throws Exception {
+        inputParameters.setForceImport("false");
+        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
+        assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
+    }
```

Comment:
Refactor the content of these two methods into a common method with a boolean parameter?

Reply:
Given it is only 2 occurrences of very short methods, and they test the 2 possible paths separately, I would prefer to keep them separate.

Reply:
In fact, the 2 tests should have additional asserts to check that different steps have been executed in each path. There should be examples in eva-pipeline.